content/docs/rendering-elements.md

```diff
@@ -1,75 +1,74 @@
 
-Consider this ticking clock example:
+Perhatikan contoh jam berdetak di bawah:
 
 `embed:rendering-elements/update-rendered-element.js`
 
-[](codepen://rendering-elements/update-rendered-element)
+[Coba di CodePen](codepen://rendering-elements/update-rendered-element)
 
-It calls `ReactDOM.render()` every second from a [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
+Contoh di atas memanggil `ReactDOM.render()` setiap detiknya dari sebuah *callback* [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval).
 
->**Note:**
+>**Catatan:**
 >
->In practice, most React apps only call `ReactDOM.render()` once. In the next sections we will learn how such code gets encapsulated into [stateful components](/docs/state-and-lifecycle.html).
+>Dalam prakteknya, sebagian besar aplikasi React hanya memanggil `ReactDOM.render()` sekali saja. Pada bagian berikutnya kita akan mempelajari bagaimana kode ini dapat dienkapsulasi ke dalam [*stateful components*](/docs/state-and-lifecycle.html).
 >
->We recommend that you don't skip topics because they build on each other.
+>Kami menyarankan agar anda tidak melewati satu topik karena topik-topik ini berhubungan satu dengan lainnya.
 
-## React Only Updates What's Necessary {#react-only-updates-whats-necessary}
+## React Hanya Memperbarui Apa yang Diperlukan {#react-only-updates-whats-necessary}
 
-React DOM compares the element and its children to the previous one, and only applies the DOM updates necessary to bring the DOM to the desired state.
+React DOM membandingkan antara elemen dan elemen anaknya dengan elemen sebelumnya, dan hanya mengaplikasikan perbaruan DOM yang diperlukan untuk menyelaraskan DOM ke *state* yang diinginkan.
 
-You can verify by inspecting the [last example](codepen://rendering-elements/update-rendered-element) with the browser tools:
+Anda dapat memverifikasi hal ini dengan menginspeksi [contoh terakhir](codepen://rendering-elements/update-rendered-element) dengan peralatan dari *browser*:
 
-![DOM inspector showing granular updates](../images/docs/granular-dom-updates.gif)
+![Inspektor DOM menunjukkan perbaruan yang bersifat granular](../images/docs/granular-dom-updates.gif)
 
-Even though we create an element describing the whole UI tree on every tick, only the text node whose contents has changed gets updated by React DOM.
+Walaupun kita membuat sebuah elemen yang mendeskripsikan struktur antarmuka pengguna secara keseluruhan dalam setiap detiknya, hanya node teks yang kontennya mengalami perubahanlah yang diperbarui oleh React DOM.
 
-In our experience, thinking about how the UI should look at any given moment rather than how to change it over time eliminates a whole class of bugs.
+Menurut pengalaman kami, memikirkan bagaimana antarmuka pengguna seharusnya terlihat pada saat tertentu daripada bagaimana mengubahnya dari waktu ke waktu dapat mengeliminasi banyak sekali *bug*.
```

Comment:
Menurut pengalaman kami, jika Anda memikirkan bagaimana antarmuka pengguna seharusnya terlihat daripada bagaimana mengubahnya dari waktu ke waktu dapat mengeliminasi banyak *bugs*.

Not quite sure either with *memikirkan* here.

Reply:
Hi @spondbob thank you for the review, for this one how about we change _memikirkan_ to _berpikir tentang_ or _berfokus pada_ ?

> Menurut pengalaman kami, jika Anda memikirkan bagaimana antarmuka pengguna seharusnya terlihat daripada bagaimana mengubahnya dari waktu ke waktu dapat mengeliminasi banyak _bugs_.
> 
> Not quite sure either with _memikirkan_ here.

Reply:
Hey that sounds good! Either one can easily fit in.

You don't have to accept my suggestion but my note would be on "at any given moment" shouldn't be "pada saat tertentu", instead "pada saat kapanpun" (anytime). But I feel like it doesn't flow well in Bahasa so I strip it out.

Otherwise good work 👍

Reply:
Thanks @spondbob 👍